prefect_utils/vault_secrets.py

```diff
@@ -39,6 +39,9 @@ def load_data_into_snowflake(sf_credentials):
 VAULT_LOGIN_URL = VAULT_BASE_URL + "/v1/auth/kubernetes/login"
 VAULT_ROLE = "prefect"
 
+# Global configuration for accessing vault from Prefect flows _outside_ of K8s.
+EXTERNAL_VAULT_BASE_URL = "https://vault.analytics.edx.org"
```

Comment:
I get that we just created this repo to be public because it would be technically easier, but everything else is already agnostic of the organization and at least I had hoped this might become useful to others.  Maybe we just pull from an environment variable, and set the variable in a makefile in the private repo, or as part of the docs: https://openedx.atlassian.net/wiki/spaces/DE/pages/1448673295/How+to+develop+prefect+flows#Local-Testing-Against-Production ?

Reply:
meh, i guess `analytics-vault.vault:8200` is already an `edx-internal`ism, so I don't want to block.  It's just a bit of a slippery slope.